packages/ui-extensions/src/surfaces/checkout/components/shared.ts

```diff
@@ -503,11 +503,28 @@ export type Size =
 
 export type Spacing =
   | 'none'
-  | 'extraTight'
-  | 'tight'
+  | 'small500'
+  | 'small400'
+  | 'small300'
+  | 'small200'
+  | 'small100'
   | 'base'
-  | 'loose'
-  | 'extraLoose';
+  | 'large100'
+  | 'large200'
+  | 'large300'
+  | 'large400'
+  | 'large500'
+  | SpacingDeprecated;
+
+/** @deprecated These values are deprecated and will eventually be removed.
+ * Use the new values.
+ *
+ * `extraTight`: `small400`
+ * `tight`: `small200`
+ * `loose`: `large200`
+ * `extraLoose`: `large500`
+ */
+export type SpacingDeprecated = 'extraTight' | 'tight' | 'loose' | 'extraLoose';
```

Comment:
Hum, I'm not sure about that.

This ended up being merged, not sure when, and we need to remove it, since the correct token changes will come later on.

cc: @jffortier

Reply:
yeah, in the near future the spacing tokes will be different than these ones that are being added here, so we will be adding this now and deprecating them, so that's not what we want now :( 

I can work on removing it from here in a bit